### DISCUSSION TITLE
z/OS Open Source Guild Session - Editor Special

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#857] [Guild]
We're excited to announce an upcoming Guild Meeting Session (with focus on TUI editors) featuring @MikeFultonDev, @HarithaIBM, and myself as we share the latest updates and upcoming features for z/OS Open Tools editors, including Vim, Nano, and Emacs.

**What to Expect:**

- :sparkles: Editor Demos: Discover the newly implemented dataset I/O capabilities in Vim, Nano, and Emacs!
- :dart: Maximizing Vim: Learn how to leverage the various Vim plugins to improve your dev workflow.
- :mag: Libdio Library: Learn about the Libdio library that powers the dataset I/O support.

You can register here: https://events.teams.microsoft.com/event/ba659f66-f995-402d-bee4-fb8058f19ac9@fcf67057-50c9-4ad4-98f3-ffca64add9e9 . 

The meeting will be held on Wed. Sept. 11 at 11am EST. As always, we appreciate your feedback and look forward to seeing you there!

Recording: https://ibm.box.com/s/yq3wmuq5brgjyxitd7zfoavf6s2ezmnp

**Comment by IgorTodorovskiIBM** [comment]
Recording: https://ibm.box.com/s/yq3wmuq5brgjyxitd7zfoavf6s2ezmnp
Powerpoint: [Open_Source_Guild_Editor_final.pptx](https://github.com/user-attachments/files/16970105/Open_Source_Guild_Editor_final.pptx)

**Comment by mgrossmann** [comment]
Great meeting yesterday. Thanks.

I have one question regarding the tagbar plugin.
I can't get it running with ctags from zot and tagbar plugin from
preservim/tagbar.

Maybe @HarithaIBM can help. :)

Thanks a lot, Mike

**Reply by HarithaIBM** [reply]
Hi @mgrossmann , do you see a blank panel when you run` :TagbarToggle` in vim?

**Reply by mgrossmann** [reply]
Exactly ....

Months ago, I could track down the problem to invalid argument handling when ctags got called. But it might be another problem, now.

**Reply by mgrossmann** [reply]
I found following problems:

```
2024-09-12 09:35:10.051003: Calling shell to execute: "('ctags' '--extras=+F' '-f' '-' '--format=2' '--excmd=pattern' '--fields=nksSafet' '--sort=no' '--append=no' '-V' '--language-force=c' '--c-kinds=hdpgetsumvf' '/tmp/v235961/10.c')>/tmp/v235961/11 2>&1"
2024-09-12 09:35:10.051512: Exit code: 1
2024-09-12 09:35:10.071392: Command output:
2024-09-12 09:35:10.071641:   Option: --language-force=c
  Option: --c-kinds=hdpgetsumvf
```

and

`ctags: cannot open temporary file: /jenkins/PortBuild/tmp/tags.deeDxQ : EDC5129I No such file or directory.`

**Reply by mgrossmann** [reply]
Any hint, @HarithaIBM ? :D 

Thanks

**Reply by HarithaIBM** [reply]
Hi @mgrossmann I have a worka round for now. Can you please try setting `TMPDIR`
For instance :

```
mkdir -p /home/local/tmp
export TMPDIR=/home/local/tmp
```

Try relaunching editor and Tagbar after this

**Reply by mgrossmann** [reply]
That works.. Ty @HarithaIBM
